packages/core/src/civil.ts

```diff
@@ -225,11 +225,11 @@ export class Civil {
   /**
    * Stores content on a content provider (defaults to IPFS)
    * @param content The the data that you want to store, in the future, probably a JSON
-   * @returns A URI that points to the content
+   * @param options Options to be passed to the provider
+   * @returns StorageHeader with info about published content
    */
-  public async publishContent(content: string): Promise<Uri> {
-    const { uri } = await this.contentProvider.put(content);
-    return uri;
+  public async publishContent(content: string, options: object): Promise<StorageHeader> {
```

Comment:
This isn't used anywhere in our codebase so should be safe to change the function signature, this way probably makes more sense anyway.

Reply:
I changed the return value so that this could be used via the `civil` instance to publish the charter in sign constitution step, as a fallback in case the component instantiating the newsroom component doesn't pass in an ipfs provider. Here's the exact bit of code: https://github.com/joincivil/Civil/commit/c2d8a558fdfcb1cf9e6bc8e54a3faed824c71f93#diff-5a61d225804b30fd4c7b30222d0dc35d